rundeckapp/grails-app/controllers/rundeck/controllers/ApiController.groovy

```diff
@@ -211,46 +213,91 @@ class ApiController extends ControllerBase{
 
 
 
+
+
+    @Tag(name = "system")
+    /**
+     * API endpoint to query system features' toggle status: True/False for On/Off
+     *
+     * The URL is `/feature/{featureName}` where featureName is the specific name of the feature without `rundeck.feature.` prefix and `enabled` surfix.
+     * E.g. The configuration item name of the feature to enable runner is `rundeck.feature.runner.enabled`, to query the status of this feature
+     *      the request will be `/feature/runner`. The result of this query is a JSON object { name: "$featureName", "enabled": true/false }
+     *
+     */
+    @Get(
+        uri= "/feature/{featureName}",
+        produces = MediaType.APPLICATION_JSON
+    )
     @Operation(
         method = "GET",
         summary = "Get Rundeck System Feature Status",
-        description = "Return feature's on/off status",
+        description = "The result of this query is a JSON object { name: \"featureName\", \"enabled\": true/false }",
```

Comment:
you don't need to outlay the json format here, it will be represented in the response schema generation